source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,65 @@
+.. _ruby-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Edit your {+language+} application file
+
+      Navigate to your ``quickstart.rb`` file in the ``ruby-quickstart``
+      directory. Copy and paste the following code under the Bundler
+      code from the :ref:`ruby-quick-start-download-and-install` step 
+      of this tutorial to query the ``movies`` collection in the ``sample_mflix``
+      database:
```

Comment:
S: Split this last sentence since it's a bit long and hard to follow
```suggestion
      directory. Copy and paste the following code below the Bundler
      code from the :ref:`ruby-quick-start-download-and-install` step 
      of this tutorial. This code connects to MongoDB and queries the 
      ``movies`` collection in the ``sample_mflix`` database.
```